packages/backend/src/handlers/actions/guild/saveGuildHandler.ts

```diff
@@ -149,4 +177,21 @@ const saveGuild = async (playerId: string, guildInfo: GuildInfoInput) => {
     guildId: guildInfo.uuid,
     discordMetadata: updatedMetadata,
   });
+
+  const { guild } = await client.GetGuild({ id: guildInfo.uuid });
+  return guild[0];
 };
+
+async function sendDiscordNotification(
+  guildInfo: GuildInfoInput,
+): Promise<void> {
+  const discordClient = await createDiscordClient();
+
+  const targetChannel = (await discordClient.channels.fetch(
+    Constants.METAFAM_DISCORD_CHAMPS_RING_CHANNEL_ID,
+  )) as TextChannel;
+  const link = `${CONFIG.hasuraAdminURL}/data/schema/public/tables/guild/browse?filter=guildname%3B%24eq%3B${guildInfo.guildname}`;
```

Comment:
Just to be extra careful about the input *(an `&`, for example, I'm pretty sure would mess up the parse)*:

```suggestion
  const link = `${CONFIG.hasuraAdminURL}/data/schema/public/tables/guild/browse?filter=${encodeURIComponent(`guildname;$eq;${guildInfo.guildname}`)}`;
```

Reply:
ah yes, good call